test/dependency-check.js

```diff
@@ -60,7 +60,7 @@ describe('dependency check', () => {
     ).to.eventually.be.rejectedWith('execa')
   })
 
-  it('should pass for passed files', async () => {
+  it.skip('should pass for passed files', async () => {
```

Comment:
There was a regression here and this is failing in #master with:

```
AssertionError: expected promise to be fulfilled but it was rejected with 'Error: Command failed with exit code 1: /Users/vsantos/gh/tmp/aegir/cli.js dependency-check derp/foo.js\n- Checking dependencies\n✖ Checking dependencies\nCommand failed with exit code 1: dependency-check package.json .aegir.js .aegir.cjs src/**/*.js src/**/*.cjs test/**/*.js test/**/*.cjs benchmarks/**/*.js benchmarks/**/*.cjs utils/**/*.js utils/**/*.cjs !./test/fixtures/**/*.js !./test/fixtures/**/*.cjs derp/foo.js --missing -i @types/* -i @types/*\nFail! Dependencies not listed in package.json: pico'
```

I skipped this for now, but I will open an issue to get this fixed

Reply:
Is this failing in master?  Passes for me locally and CI for master is green?

The test should pass because it's only supposed to check the passed file, but if it's failing it means it's checking more than just the passed file - it's probably worth looking into why this is now broken.

Reply:
Yes, it is failing in master too. Did you update the package lock?
Unless it got fixed in the meantime, let me try again

Reply:
```
➜  aegir git:(master) nrt

> aegir@34.1.0 test
> node cli.js ts -p check && node cli.js test

Test Node.js


  dependency check
    1) should pass for passed files


  0 passing (520ms)
  1 failing

  1) dependency check
       should pass for passed files:
     AssertionError: expected promise to be fulfilled but it was rejected with 'Error: Command failed with exit code 1: /Users/vsantos/work/pl/gh/tmp/aegir/cli.js dependency-check derp/foo.js\n- Checking dependencies\n✖ Checking dependencies\nCommand failed with exit code 1: dependency-check package.json .aegir.js .aegir.cjs src/**/*.js src/**/*.cjs test/**/*.js test/**/*.cjs benchmarks/**/*.js benchmarks/**/*.cjs utils/**/*.js utils/**/*.cjs !./test/fixtures/**/*.js !./test/fixtures/**/*.cjs derp/foo.js --missing -i @types/* -i @types/*\nFail! Dependencies not listed in package.json: pico'
```

I will reinstall all deps

Reply:
Actually yeah, removing the lockfile and reinstalling now it's failing. Ugh.

Reply:
also FYI this happened without any changes in this PR, except for the docs being added. The reason to ignore this here was because this was not related to the PR and I did not want to keep this blocked on an unrelated #master problem

Reply:
(still happens for me in #master) by removing the package-lock and installing everything with Node 16)

Reply:
I think yargs has started merging passed array args with the defaults for that positional. Fix: https://github.com/yargs/yargs/pull/2006

Reply:
yeah, that sounds like the problem when I was trying to figure it out. Can we move forward with the PR and wait on that to be shipped to unskip the tests?